pkg/pss/utils/mapping.go

```diff
@@ -110,18 +112,20 @@ var PSS_control_name_to_ids = map[string][]string{
 	},
 }
 
-// reverse mapping of PSS_control_name_to_ids
-var pss_control_id_to_name = map[string]string{}
-
-func PSSControlIDToName(id string) string {
-	if len(pss_control_id_to_name) == 0 {
-		for name, ids := range PSS_control_name_to_ids {
-			for _, id := range ids {
-				pss_control_id_to_name[id] = name
-			}
+// initialize reverse mapping of PSS_control_name_to_ids
+func initPSSControlNameToIdsMapping() map[string]string {
+	pss_control_id_to_name := make(map[string]string)
+	for name, ids := range PSS_control_name_to_ids {
+		for _, id := range ids {
+			pss_control_id_to_name[id] = name
 		}
 	}
-	return pss_control_id_to_name[id]
+	return pss_control_id_to_name
+}
+
+func PSSControlIDToName(id string) string {
+	pss_control_id_to_name := sync.OnceValue(initPSSControlNameToIdsMapping)
```

Comment:
it doesn't make sense tu use `sync.OnceValue` in a local variable ;-)

Reply:
My bad, thanks for pointing that out :)